/aruco_node:
    ros__parameters:
        # Aruco detection parameters
        marker_size: 0.2 # Size of the markers in meters
        aruco_dictionary_id: DICT_7X7_50 # Aruco dictionary type
        image_topic: /camera/color/image_raw # Input image topic
        camera_info_topic: /camera/color/camera_info # Input camera info topic with camera intrinsic parameters
        camera_frame: camera_color_optical_frame # Camera link frame of reference
        # Output topics
        detected_markers_topic: /aruco/markers # Output topic with detected markers (aruco poses + ids)
        markers_visualization_topic: /aruco/poses # Output topic with visualization of detected markers as pose array
        display_image_topic: /aruco/image # Output topic with visualization of detected markers drawn on image
